Assets/LeapMotion/Modules/InteractionEngine/Scripts/InteractionBehaviour.cs

```diff
@@ -803,10 +811,41 @@ public enum GraspedMovementType {
     public void FixedUpdateObject() {
       if (!ignoreGrasping) fixedUpdateGrasping();
       fixedUpdateLayers();
+      fixedUpdatePose();
 
       if (_appliedForces) { FixedUpdateForces(); }
     }
 
+    #region Pose & Movement
+
+    private Pose _worldPose;
+    private Pose _worldPoseLastFrame = new Pose();
+    private bool _hasWorldPoseLastFrame = false;
+
+    public Pose worldPose {
+      get {
+        return _worldPose;
```

Comment:
Any reason this doesn't just construct the pose on the spot?  If the user moves a rigidbody manually it will become stale.

Reply:
good idea